src/index.ts

```diff
@@ -1,38 +1,14 @@
-import libWordArray from "crypto-js/lib-typedarrays";
-import SHA256 from "crypto-js/sha256";
-import encBase64url from "crypto-js/enc-base64url";
+import type { webcrypto } from 'node:crypto';
 
-/**
- * Thanks to @SEIAROTg on stackoverflow:
- * "Convert a 32bit integer into 4 bytes of data in javascript"
- * @param num The 32bit integer
- * @returns An ArrayBuffer representing 4 bytes of binary data
- */
-function toBytesInt32(num: number) {
-  const arr = new ArrayBuffer(4); // an Int32 takes 4 bytes
-  const view = new DataView(arr);
-  view.setUint32(0, num, false); // byteOffset = 0; litteEndian = false
-  return arr;
-}
+const crypto: webcrypto.Crypto = globalThis.crypto ?? (await import('node:crypto')).webcrypto;
```

Comment:
Make sure that the code can run in Node.js v16 REPL.
```suggestion
const crypto: webcrypto.Crypto =
  globalThis.crypto?.webcrypto ??
  globalThis.crypto ??
  (await import("node:crypto")).webcrypto;
```

Reply:
```suggestion
const crypto: webcrypto.Crypto =
  globalThis.crypto?.webcrypto ?? // Node.js 16 REPL has globalThis.crypto as node:crypto
  globalThis.crypto ?? // web browsers and Node.js 18+ 
  (await import("node:crypto")).webcrypto; // Node.js 16 non-REPL
```